userdocs/src/usage/eks-managed-nodes.md

```diff
@@ -184,6 +185,11 @@ To upgrade to a specific AMI release version instead of the latest version, pass
 eksctl upgrade nodegroup --name=managed-ng-1 --cluster=managed-cluster --release-version=1.19.6-20210310
 ```
 
+## Handling parallel upgrades for nodes
+
+Multiple managed nodes can be upgraded simultaneously. `upgradeConfig` can be used to handle parallel upgrades. To avoid any downtime to your workloads due to upgrading multiple nodes at once, limit the number of nodes that can become unavailable during an upgrade as `maxnavailable`. Alternatively, use `maxUnavailablePercentage`, which defines the maximum number of unavailable nodes as a percentage of the total number of nodes. Note that this feature is only available for managed nodes.
```

Comment:
should this be `updateConfig`?

(I mean it _should_ be upgrade, but I think we/they have called it update like savages 😉 )

Reply:
good spot! and yep ++ agreed 👀 

the user docs needed some extra TLC: https://github.com/weaveworks/eksctl/pull/3753/commits/a58cf6ff7ea98c39cef56f094a3ecf1744a0f771